aws/acm.go

```diff
@@ -34,20 +36,47 @@ func (p *acmCertificateProvider) GetCertificates() ([]*certs.CertificateSummary,
 	return result, nil
 }
 
-func getACMCertificateSummaries(api acmiface.ACMAPI) ([]*acm.CertificateSummary, error) {
+func getACMCertificateSummaries(api acmiface.ACMAPI, filterTag string) ([]*acm.CertificateSummary, error) {
 	params := &acm.ListCertificatesInput{
 		CertificateStatuses: []*string{
 			aws.String(acm.CertificateStatusIssued),
 		},
 	}
 	acmSummaries := make([]*acm.CertificateSummary, 0)
+
 	err := api.ListCertificatesPages(params, func(page *acm.ListCertificatesOutput, lastPage bool) bool {
 		acmSummaries = append(acmSummaries, page.CertificateSummaryList...)
 		return true
 	})
+
+	if tag := strings.Split(filterTag, "="); len(tag) == 2 {
```

Comment:
if filterTag = "=" this condition will also work.

Reply:
Is this case possible?

Reply:
Maybe we don't need to handle I'm just mentioning the case

Reply:
True, I'll fix that.

Reply:
What is the comment about @MustafaSaber ?
I don't understand your suggestion.

Reply:
It's just an edge case, that someone will set filterTag to be `"="`

Reply:
ah got it, but then you have not 2 and it's fine to not support `=` as filter tag nor as value